Tests/Unit/Repository/PhpcrOdmRepositoryTest.php

```diff
@@ -138,4 +143,98 @@ public function testGetVersionsWillThrow()
 
         $this->getRepository()->getVersions('/test');
     }
+
+    /**
+     * @dataProvider provideAddInvalid
+     *
+     * @expectedException \InvalidArgumentException
```

Comment:
but running them all throug the dataprovider?
The messages are very differnt i think, but you are right, i need to be sure, that i match the right one.